StortSpelProjekt/Engine/src/Renderer/Renderer.h

```diff
@@ -113,19 +115,10 @@ class Renderer
 
 	SwapChain* GetSwapChain();
 
-	// Load Gpu Memory Functions
-	void LoadModel(Model* model) const;
-	void LoadMesh(Mesh* mesh) const;
-	void LoadMaterial(Material* material) const;
-	void LoadTexture(Texture* texture) const;
-
-	// Unload Gpu Memory Functions
-	void UnloadModel(Model* model) const;
-	void UnloadMesh(Mesh* mesh) const;
-	void UnloadMaterial(Material* material) const;
-	void UnloadTexture(Texture* texture) const;
-
-	void UnloadRenderComponents();
+	void SubmitToCodt(std::tuple<Resource*, Resource*, const void*>* Upload_Default_Data);
+	void SubmitModelToCodt(Model* model);
+	void SubmitMeshToCodt(Mesh* mesh);
+	void SubmitTextureToCodt(Texture* texture);
```

Comment:
If these are public, are they meant to be called by game-developer, or is it a misstake by you? If they are meant to be called by the game-developer. Then they would also need a function to execute the commandlist after data has been submitted.

Reply:
moved to private